src/Orleans/Serialization/SerializationTestEnvironment.cs

```diff
@@ -0,0 +1,34 @@
+namespace Orleans.Serialization
+{
+    using System.Collections.Generic;
+    using System.Reflection;
+
+    using Orleans.Runtime;
+
+    public class SerializationTestEnvironment
```

Comment:
Explained in the PR description, it's just to make it easier to initialize SerializationManager given some parts which were previously static are no longer static.

I hope to get rid of most of this, but in a later PR where SerializationManager is no longer static.